internal/testplanet/uplink_test.go

```diff
@@ -98,9 +98,6 @@ func TestDownloadWithSomeNodesOffline(t *testing.T) {
 			if nodesToKill[node.ID()] {
 				err = planet.StopPeer(node)
 				require.NoError(t, err)
-
-				err = satellite.Overlay.Service.Delete(ctx, node.ID())
```

Comment:
I'm wondering whether we should make a function to mark nodes offline or the last ping failing instead.

Reply:
Good point!

Reply:
Fixed with the latest commit.